src/js/classes/Map.js

```diff
@@ -0,0 +1,680 @@
+import * as d3 from "d3";
+import { feature } from "topojson";
+import { COUNTY_MAP_DATA } from "../data.js";
+
+// TODO: Support mobile sizings
+const MAP_WIDTH = 600;
```

Comment:
Similar to a comment I added to Alex's PR. I wonder if we can handle responsiveness using 100% instead of a fixed number. Seems like there should be a way in D3 to have a fluid container

Reply:
Ended up using a viewBox, please let me know if I'm doing something dumb

Reply:
If at some point we wanna make the viewboxes fully responsive, it can be a seperate PR.

Reply:
https://github.com/aclu-cash-bail-team/aclu-cash-bail/issues/45